doc/POST_INSTALL.md

```diff
@@ -0,0 +1,15 @@
+If your server name is identical to the domain on which synapse is installed, and the default port 8448 is used, your server is normally already accessible by the federation.
+
+If not, you may need to put the following line in the dns configuration:
+
+_matrix._tcp.__DOMAIN__. 3600    IN      SRV     10 0 __PORT_SYNAPSE_TLS__ __DOMAIN__.
```

Comment:
````suggestion
```text
_matrix._tcp.__DOMAIN__. 3600    IN      SRV     10 0 __PORT_SYNAPSE_TLS__ __DOMAIN__.
```
````